tools/dvrescue.xsd

```diff
@@ -176,6 +177,18 @@
     </xsd:restriction>
   </xsd:simpleType>
 
+  <xsd:simpleType name="captionsType">
+    <xsd:annotation>
+      <xsd:documentation xml:lang="en">
+        Values of 'y' and 'n' indicate whether there is or is not the presence of EIA-608 closed captioning packs with the VAUX section of the DV frames.
```

Comment:
'n' is not implemented, it is 'y' if detected else not present, so it should be indicated that is not present it means that it was not detected (similar to 'n').

Reply:
On line 95, it defines that 'n' is default, so when no @captions attribute present it is semantically identical to @captions='n'.